### DISCUSSION TITLE
how to use r export data

### DISCUSSION THREAD

**Discussion by Jerryblue2030** [jamesjiadazhan/dietaryindex#246] [Q&A] [answered]
Hello, brother.  I have finished the calculation of DII, but how to export the data? 
![微信图片_20240117200630](https://github.com/jamesjiadazhan/dietaryindex/assets/156912599/089a984b-0cb5-4bf8-b5e1-a87ba6154587)

**Comment by wangdsds** [comment]
Brother, how did you come up with this result? Can I request code

**Comment by jamesjiadazhan** [comment, marked as answer]
Please check the last section of the manual:
https://jamesjiadazhan.github.io/dietaryindex_manual/

Remember, if you ever use DII for NHANES, please include weighting variables in your analysis. This includes the consideration of the complex survey design used in NHANES, which is necessary to produce valid data for publication.
